src/core_modules/capture-core/components/Pages/MainPage/WorkingLists/TemplateMaintenance/SharingDialog.component.js

```diff
@@ -1,35 +1,42 @@
 // @flow
-import * as React from 'react';
+import React, { useCallback, type ComponentType } from 'react';
 import { withStyles } from '@material-ui/core/styles';
 import D2UISharingDialog from '@dhis2/d2-ui-sharing-dialog';
 import { getD2 } from '../../../../../d2';
+import type { Props } from './sharingDialog.types';
 
-const getStyles = () => ({
+const styles = {
```

Comment:
I always thought this had to be a function. Good to know it isnt the case

Reply:
I actually thought I picked this up from you 🙂

Reply:
Maybe, I think I always use it as a function though. Perhaps I am forgetful